supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,145 @@
+[[openshift-container-storage-conventions]]
+
+[discrete]
+[[backing-store]]
+==== backing store (noun)
+*Description*: A type of storage resource for Multicloud Object Gateway, which is used to store the data like RADOS gateway (RGW), Amazon Web Services S3, Azure Blob Storage, IBM Cloud Object Storage, and so on. 
```

Comment:
Just wondering if we should remove 'the' in `..which is used to store the data like..`? because it doesn't scan very well this way. Maybe another suggestion could be:

`.., which is used to store data, for example, from RADOS gateway (RGW), Amazon Web Services S3, Azure Blob Storage, IBM Cloud Object Storage.`

the 'for example' would remove the need for "and so on".

Reply:
Changed.